shared/util.js

```diff
@@ -0,0 +1,332 @@
+/**
+ * @license
+ * Copyright 2017 The Lighthouse Authors. All Rights Reserved.
```

Comment:
Should we update this license?

Reply:
I'm gonna go on a limb and say these comments don't matter. ever